main.go

```diff
@@ -64,18 +73,40 @@ func parseProblems(problems [][]string) []QAPair {
 func compAns(arrStruct []QAPair) {
 	correctAns := 0
 	incorrectAns := 0
+
+	timeoutChannel := make(chan bool, 1)
+	nameChannel := make(chan bool, 1)
```

Comment:
Naming this answerChannel might be a little more intuitive.
```suggestion
	answerChannel := make(chan bool, 1)
```